src/test/java/de/tobiasroeser/lambdatest/proxy/ExampleProxyTest.java

```diff
@@ -107,22 +118,79 @@ public boolean genericArg(String arg1) {
 				}
 			});
 			intercept(UnsupportedOperationException.class,
-					"(?s).*\\Qpublic boolean genericArg(Object object1)\\E.*",
+					"(?s).*\\Qpublic boolean genericArg(T x0)\\E.*",
```

Comment:
I don't think this "suggestion" will work for the implementer of the proxy delegate object, as T isn't bound to anything in the delegate object. The previous `Object object1` is much better.